code/datums/uplink/devices and tools.dm

```diff
@@ -54,7 +54,7 @@
 
 /datum/uplink_item/item/tools/heavy_helmet
 	name = "Heavy Armor Helmet"
-	item_cost = 4
+	item_cost = 6
```

Comment:
The helmet is really good but too cheap, the vest is extremely underused, because it isn't that good. This makes the price reflect how good they are, and how much they are used.